### PR TITLE
Update service name for networkcloud

### PR DESCRIPTION
**Pull request by Saisang**

### CODE REVIEW

**Comment by Saisang** [comment]
#sign-off

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -202,7 +202,7 @@
 "Azure.ResourceManager.NetApp","1.1.0","1.2.0-beta.1","Resource Management - NetApp Files","NetApp Files","netapp","","","mgmt","true","","","09/28/2022","","","","","","","",""
 "Azure.ResourceManager.Network","1.2.0","1.3.0-beta.1","Resource Management - Network","Network","network","","","mgmt","true","","","07/11/2022","","","","","","","",""
 "Azure.ResourceManager.NetworkFunction","","1.0.0-beta.3","Resource Management - Network Function","Network Function","networkfunction","","","mgmt","true","","","","","","","","","","",""
-"Azure.ResourceManager.NetworkCloud","","1.0.0-beta.1","Resource Management - Networkcloud","Networkcloud","networkcloud","","","mgmt","true","","","","","","","","","","","Needs Review"
+"Azure.ResourceManager.NetworkCloud","","1.0.0-beta.1","Resource Management - Networkcloud","Azure Operator Nexus - Network Cloud","networkcloud","","","mgmt","true","","","","","","","","","","","Needs Review"
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:205]: We typically don't include Microsoft or Azure in the name fields as those are generally implied in the context we use these.

Comment (Saisang) [reply _data/releases/latest/dotnet-packages.csv:205]: The content owner expected to customize the toc name as this. I'll let them answer here.

Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:205]: @lfraleigh @ronniegeraghty I'll defer to you about the best name here.

**Comment by rashirg** [comment]
@Saisang as suggested please update the name with "Operator Nexus - Network Cloud" for all the Languages requested. Thank you

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (ronniegeraghty) [reply _data/releases/latest/dotnet-packages.csv]: From the spec I'd expect the name to be just Cloud Network, but I understand if the team want's it to show a certain way in the ToC. As long as the name is consistent across languages it should be fine. 
@Saisang, can you break down the `Operator Nexus - Network Cloud` name for me. Is `Operator Nexus` the service name and is `Network Cloud` an offering of that service? Or is the whole service name `Operator Nexus - Network Cloud`?

Comment (Saisang) [reply _data/releases/latest/dotnet-packages.csv]: @ronniegeraghty The whole name Operator Nexus - Network Cloud. Can't the dash be in the name? Should I use escape character or else?

Comment (ronniegeraghty) [reply _data/releases/latest/dotnet-packages.csv]: @Saisang, the name can have the dash. I just wanted to know a little more about how this maps to Azure services, products, and offerings. If the service name is `Operator Nexus - Network Cloud` then that should be in the `Service` column and the `Display Name` column should be `Resource Management - Network Cloud`.

Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv]: @danieljurek won't the `-` in the service name lead to nesting in the ToC on the docs side?

Comment (danieljurek) [reply _data/releases/latest/dotnet-packages.csv]: @weshaggard -- nesting from `-` only happens when the Service is `Other` and the dashes are in the display name.

Comment (Saisang) [reply _data/releases/latest/dotnet-packages.csv]: @danieljurek Could you merge this today so we can verify it tomorrow? The next publish will be next Tuesday so we can time to revert if something wrong.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**